packages/quickswap/src/styles.ts

```diff
@@ -502,15 +530,15 @@ export const configureStyles = (isProMode?: boolean) => {
           background: "#3E4252",
         },
         "& .MuiButtonBase-root": {
-          color: styles.orderHistoryTabColor,
+          color: isLightMode ? styles.textColor : styles.orderHistoryTabColor,
```

Comment:
maybe give the same style

color:  styles.textColor 

and change the value in the styles object itself